composer.json

```diff
@@ -23,5 +23,8 @@
         "mozilla/wp-sw-manager": "dev-master",
         "marco-c/wp_serve_file": "dev-master"
     },
+    "autoload": {
+      "classmap": ["vendor/"]
```

Comment:
no thats not correct... here you should define where "your" classes are placed, not those of your dependencies